vocab/en-us/GetKeyword.voc

```diff
@@ -0,0 +1,4 @@
+whats
```

Comment:
normalize() takes care of this, transforming "what's" and "whats" into "what is"